docs/content/configuration/csi_ug_config_create_replication.md

```diff
@@ -0,0 +1,46 @@
+# Creating a VolumeReplication
+
+Create a VolumeReplication yaml file to replicate a specific PersistentVolumeClaim (PVC).
+
+VolumeReplicationClass needs to be present before a VolumeSnapshot can be created. For more information, see [Creating a VolumeReplicationClass](csi_ug_config_create_vol_replicationclass.md).
+
+**Note:** Replication is also known as mirroring in many storage systems.
+
+When replicating a volume, be sure to follow all of the replication configurations, found in [Compatibility and requirements](../installation/csi_ug_requirements.md) before snapshot creation.
+
+1.  Replicate a specific  PersistentVolumeClaim (PVC) using the demo-replication.yaml.
+
+    For more information about PVC configuration, see [Creating a PersistentVolumeClaim (PVC)](csi_ug_config_create_pvc.md).
+
+    ```
+    apiVersion: replication.storage.openshift.io/v1alpha1
+    kind: VolumeReplication
+    metadata:
+      name: volumereplication-sample
+      namespace: default
+    spec:
+      volumeReplicationClass: volumereplicationclass-sample
+      replicationState: primary
+      dataSource:
+        kind: PersistentVolumeClaim
+        name: myPersistentVolumeClaim # should be in same namespace as VolumeReplication
+    ```
+
+2.  After the YAML file is created, apply it by using the `kubectl apply -f` command.
```

Comment:
why is `YAML` sometimes capitalized and sometimes not?

Reply:
according to IBM Style it should be capitalized if referred to by itself and lower case if with `.yaml` file. I'll review the different times that it's in the document and make sure that it's consistent.

Reply:
thanks

Reply:
DONE